pytest/tests/mocknet/load_test_spoon.py

```diff
@@ -131,21 +139,30 @@ def check_slow_blocks(initial_metrics, final_metrics):
     logger.info(f'initial_validator_accounts: {initial_validator_accounts}')
     test_passed = True
 
+    script, deploy_time, test_timeout = (None, None, None)
+    if args.script == 'skyward':
+        script, deploy_time, test_timeout = 'load_testing_add_and_delete_helper.py', load_testing_add_and_delete_helper.CONTRACT_DEPLOY_TIME, load_testing_add_and_delete_helper.TEST_TIMEOUT
```

Comment:
Nah, just split it (and the assignment below) into three separate lines. Easier to read that way.